tests/nlu/extractors/test_regex_entity_extractor.py

```diff
@@ -86,12 +86,17 @@ def test_process(
     training_data.lookup_tables = lookup
     training_data.training_examples = [
         Message(
-            data={TEXT: "Hi Max!", "entities": [{"entity": "person", "value": "Max"}]}
+            data={
+                TEXT: "Hi Max!",
+                INTENT: "intent",
```

Comment:
why do we need dict entries set to `"intent"`?

Reply:
The `RegexEntityExtractor` uses `training_data.nlu_examples`. We define an NLU example as an training example, that has an intent and a name set.

Reply:
should we rather use a value that indicates that this is an intent then, e.g. `greet`?

Reply:
you mean in the test? sure we could update the value from `intent` to `greet`

Reply:
yes, it'd be great if you can do that at the other places as well. I don't think it helps readability if a value is set to `intent` or `text`, for me an actual example is way easier to digest